scripts/2500/stage/NIPT/syncnipt.bash

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# script to rsync a run to the NIPT server
+
+shopt -s nullglob
+set -eu
+
+VERSION=5.4.2
+echo "Version $VERSION"
+
+##########
+# PARAMS #
+##########
+
+RUNBASE=/home/proj/stage/flowcells/2500/runs/
+NIPTBASE=/home/proj/stage/flowcells/2500/nipt/
+NIPTOUTPATH=/home/proj/stage/flowcells/2500/srv/nipt_runs/
+MAILTO_ERR=YOUR.NAME@scilifelab.se
+
+#############
+# FUNCTIONS #
+#############
+
+failed() {
+    echo "Fail to sync ${RUN}" | mail -s "ERROR syncing NIPT $(hostname):${RUN}" ${MAILTO_ERR}
+}
+trap failed ERR
+
+#######
+# RUN #
+#######
+
+for RUN in ${RUNBASE}/*; do
+  RUN=$(basename ${RUN})
+  NOW=$(date +"%Y%m%d%H%M%S")
+  if [[ ! -e ${NIPTBASE}${RUN} ]]; then
+    # simple NIPT detection
+    if grep -qs Description,cfDNAHiSeqv1.0 ${RUNBASE}${RUN}/SampleSheet.csv; then
+      if [[ ! -e ${RUNBASE}${RUN}/SampleSheet.ori ]]; then
+        cp ${RUNBASE}${RUN}/SampleSheet.csv ${RUNBASE}${RUN}/SampleSheet.ori
+      fi
+
+      # transform SampleSheet from Mac/Windows to Unix
+      if grep -qs $'\r' ${RUNBASE}${RUN}/SampleSheet.csv; then
+          sed -i 's/
+/\n/g' ${RUNBASE}${RUN}/SampleSheet.csv
```

Comment:
Same here, single line?

Reply:
See eariler comment.

If you open the file it looks like this:
![image](https://user-images.githubusercontent.com/26384108/109658379-d37d0a80-7b66-11eb-9d8a-81ad877d59f5.png)

If you look at the file directly in Github it looks like this:
![image](https://user-images.githubusercontent.com/26384108/109658474-ea236180-7b66-11eb-82e1-8e0334e57133.png)

So, it is a single line!